src/libraries/Native/Unix/System.Globalization.Native/pal_locale.h

```diff
@@ -4,6 +4,9 @@
 
 #include "pal_icushim.h"
 
+// if defined we're building the qcalls entrypoints which don't include ICU headers
+#ifndef __LIB_NATIVE_ENTRYPOINTS
```

Comment:
Would it look better to spit this file into two?

Reply:
I wanted to avoid touching all the files that imported this header to import the new header. 

Or I can split this file in two and just import the new header in this header ifndef __LIB_NATIVE_ENTRYPOINTS.

Reply:
> I wanted to avoid touching all the files that imported this header 

It is trivial find&replace. I do not see that as a problem.

This header diverges from the style used by all the other headers (it is mixing the entrypoints and internal helpers). I think there is a value in uniformity.

Reply:
Ok, I'll do that. there is another one which does that as well `pal_errors.h` so I'll do it for that one also.